index.html

```diff
@@ -24,47 +28,43 @@ <h1 class="display-4">Playlist</h1>
 
     <subheader>
       <h2 class="display-6">2024 SDF 9.1</h2>
-      <!--Change Me-->
     </subheader>
-    <table class="table table-striped table-bordered">
-      <thead class="thead-dark">
-        <tr>
-          <th scope="col">Name</th>
-          <th scope="col">Artist</th>
-          <th scope="col">Song</th>
-          <th scope="col">Link</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <td>Aldo Portillo</td>
-          <td>Garth Brooks</td>
-          <td>Friends in Low Places</td>
-          <td>
-            <a
-              href="https://www.youtube.com/watch?v=ZcQeocF56Ck"
-              target="_blank"
-              class="btn btn-primary"
-            >
-              Link
+
+    <div class="list-group">
+      <div
+        class="list-group-item d-flex justify-content-between align-items-center"
+      >
+        <div>
+          <h3>Friends in Low Places</h3>
+          <h4>Garth Brooks</h4>
+          <p>
+            Submitted by
+            <a href="https://github.com/aldoportillo" target="_blank">
+              Aldo Portillo
             </a>
-          </td>
-        </tr>
-        <tr>
-          <td>Jose Montes De Oca Morfin</td>
-          <td>The Weeknd</td>
-          <td>Wanderlust</td>
-          <td>
-            <a
-              href="https://www.youtube.com/watch?v=PrCG4EK1Ehs"
-              target="_blank"
-              class="btn btn-primary"
-            >
-              Link
+          </p>
+        </div>
+        <a href="https://www.youtube.com/watch?v=ZcQeocF56Ck" target="_blank">
+          <i class="bi bi-play-circle-fill play-icon"></i>
+        </a>
+      </div>
+      <div
+        class="list-group-item d-flex justify-content-between align-items-center"
+      >
+        <div>
+          <h3>Wanderlust</h3>
+          <h4>The Weeknd</h4>
+          <p>
+            Submitted by
+            <a href="https://github.com/JoseMDO" target="_blank">
+              Jose Montes De Oca Morfin
             </a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+          </p>
+        </div>
+        <a href="https://www.youtube.com/watch?v=PrCG4EK1Ehs" target="_blank">
```

Comment:
this button is really small. maybe we can use a chevron? https://fontawesome.com/icons/chevron-right?f=classic&s=solid

Reply:
might not be necessary if we use this https://getbootstrap.com/docs/5.3/components/list-group/#custom-content